development.md

```diff
@@ -0,0 +1,156 @@
+# Development
+
+## Introduction
+
+OpenRazer consists of a Linux kernel module and Python daemon + library. The Linux kernel module exposes an
+sysfs interface, the Python daemon (openrazer-daemon) exposes this on it's turn over D-Bus.
```

Comment:
"exposes a D-Bus interface for applications to use."

Also add a sentence about pylib please, something like:

"It also includes a Python libraries for applications and scripts to use that talks to the daemon over D-Bus."

Reply:
:+1: added in 8a3761fb96a363ec2a07de1f985d92c9b301cc84